packages/passport/sdk/src/zkEvm/user/registerZkEvmUser.ts

```diff
@@ -34,9 +34,11 @@ export async function registerZkEvmUser({
   const headers = { Authorization: `Bearer ${accessToken}` };
 
   const { chainId } = await jsonRpcProvider.ready;
+  const eipChainId = getEip155ChainId(chainId);
+  const chainList = (await multiRollupApiClients.chainsApi.listChains()).data.result;
```

Comment:
Do/ How we want to handle the error

Reply:
We can probably just move it down within the existing try/catch block. Does that work for you?